docs/bot-optimization.md

```diff
@@ -0,0 +1,157 @@
+# Bot Optimization
```

Comment:
I think these should be Github wiki pages that we link to from README.md.

Reply:
The issue with the wiki is that no-one can edit it if you are not an admin. And even for us we cannot review and roll it back.

I would like to give the chance from non-tech to contribute to the doc.